mfa/errors.md

```diff
@@ -1,15 +1,239 @@
-# Errors
+{% multimethod %}
+{% endmultimethod %}
 
-Errors are based on the [Voice API Errors](../voice/errors.md) and the [Messaging API Errors](../messaging/errors/about.md).
+# HTTP Errors when using MFA {#top}
 
-## Error Responses
+Bandwidth will respond with HTTP error codes when there are any issues with . Any `4xx` or `5xx` HTTP response code will not be billed. All of the errors except for `401` and `403` have a similar response format. Unifying all of the responses is in the works, so be sure to keep checking the page for updates!
```

Comment:
The first sentence isn't complete

Reply:
Thanks for catching that, missed putting MFA